pandas/tests/io/parser/test_header.py

```diff
@@ -389,6 +389,17 @@ def test_header_multi_index_common_format_malformed3(all_parsers):
     tm.assert_frame_equal(expected, result)
 
 
+def test_header_multi_index_blank_line(all_parsers):
+    # GH 40442
+    parser = all_parsers
+    data = [[None, None], [1, 2], [3, 4]]
+    columns = MultiIndex.from_tuples([("a", "A"), ("b", "B")])
+    expected = DataFrame(data, columns=columns)
+    data = "a,b\nA,B\n,\n1,2\n3,4"
+    result = parser.read_csv(StringIO(data), header=[0, 1])
```

Comment:
what happens with `index_col` not None (do we already tests this)

Reply:
Yes, these two tests both have `MultiIndex` headers and `index_col` not None:
https://github.com/pandas-dev/pandas/blob/526d52f91ee06ba9134762373ab3a0bb38395d8a/pandas/tests/io/parser/test_header.py#L126
https://github.com/pandas-dev/pandas/blob/526d52f91ee06ba9134762373ab3a0bb38395d8a/pandas/tests/io/parser/test_header.py#L219